### DISCUSSION TITLE
v20

### DISCUSSION THREAD

**Discussion by oae** [oae/gnome-shell-pano#227] [Announcements]
### A big shout-out to @Totto16 for his fantastic job in bringing the extension over to Gnome 45!

--

##### Documentation Changes

*  update readme with fedora build dependencies. closes #149 (4a16c4f3)

##### New Features

* add support for Gnome 45 (#222) (9c8d2327)
*  add wiggle animation to indicator (01f3de8b)
*  add font customization for search and title. #104 (e9343ddb)
*  add window position settings. closes #165 (90d7a1be)
*  add icon-pack selection. closes #148 (c3eb97a8)
* **locales:**
  *  update german translation (#184) (cf871244)
  *  update ru.po (999552fa)
  *  update ru.po (#179) (17f8c5d8)
  *  update French translation (#171) (7c6e9319)
  *  update it.po (#167) (b1795d3a)
  *  update german translation (#160) (84fca888)
  *  update it.po (#153) (52f6c86d)

##### Bug Fixes

*  stop key press events from propagation on search entry (8595c790)
*  trim content when showing notification (56636d02)
*  don't trim text during copy/paste. fixes #168 (d84f03a3)
*  show/hide open link button based on the settings (d62ab51f)
*  change open link behavior. fixes #156 (b8b2836b)
* **locales:**  update german translation (#161) (85af0608)

<hr /><em>This discussion was created from the release <a href='https://github.com/oae/gnome-shell-pano/releases/tag/v20'>v20</a>.</em>

**Comment by Fabio286** [comment]
Amazing job with Gnome 45 ❤️

**Comment by hermanbossou** [comment]
Great job and thanks to @Totto16

**Comment by wendevlin** [comment]
Great job thanks.
I cannot use it in Gnome 45 when I open https://extensions.gnome.org/extension/5278/pano/ it says outdated and the last shell version is 44. Do you know how to fix this?
